File: Day26/Part5/oracle_stones.py
from typing import List

def min_final_value(a: List[int]) -> int:
    x = 0
    for v in a:
        x ^= v
    return x

if __name__ == "__main__":
    a = [1,3,5,7]
    print(min_final_value(a))  # Expected 0

File: Day26/Part5/oracle_stones.cpp
#include <bits/stdc++.h>
using namespace std;

int min_final_value(const vector<int>& a){
    int x = 0;
    for(int v: a) x ^= v;
    return x;
}

int main(){
    vector<int> a = {1,3,5,7};
    cout << min_final_value(a) << "\n"; // Expected 0
    return 0;
}
